CHANGELOG.md

```diff
@@ -2,6 +2,85 @@
 
 This file documents all notable changes to Juttle. The release numbering uses [semantic versioning](http://semver.org).
 
+## 0.5.0
+
+### Major Changes
+
+- Implemented consolidated, versioned adapter API. Adapters should use the `AdapterAPI` global variable to access the API instead of `require`-ing modules from the `juttle` package directly. [[#367](https://github.com/juttle/juttle/issues/367)]
+
+### Minor Changes
+
+- Made `reduce -every` emit marks at the end of epoch. [[#52](https://github.com/juttle/juttle/issues/52)]
```

Comment:
This seems like a major change to me since it changes the semantics of programs.